tests/test_tokenization_wav2vec2.py

```diff
@@ -554,3 +554,11 @@ def test_add_tokens_tokenizer(self):
                 self.assertGreater(tokens[-3], tokens[-4])
                 self.assertEqual(tokens[0], tokenizer.eos_token_id)
                 self.assertEqual(tokens[-3], tokenizer.pad_token_id)
+
+    @unittest.skip("The tokenizer shouldn't be used to encode input IDs (except for labels), only to decode.")
+    def test_tf_encode_plus_sent_to_model(self):
+        pass
+
+    @unittest.skip("The tokenizer shouldn't be used to encode input IDs (except for labels), only to decode.")
+    def test_pt_encode_plus_sent_to_model(self):
```

Comment:
I cannot find this function in `tests/test_tokenization_common.py` -> is it maybe spelled differently?